packages/vscode-extension/package.nls.json

```diff
@@ -393,8 +393,8 @@
     "_teamstoolkit.viewsWelcome.teamsfx-empty-project-new-user.content.comment": "For command like [Get Started](command:xxx), please translate 'Get Started' and keep the string 'command:xxx'",
     "teamstoolkit.viewsWelcome.teamsfx-feedback.content": "Take 2 minutes to help us improve, your feedback matters!\n[We Would Love Your Feedback](command:fx-extension.openSurvey)",
     "teamstoolkit.walkthroughs.description": "Jumpstart your Teams app development experience",
-    "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.description": "Choose [Teams app capabilities](https://aka.ms/teamsfx-capabilities-overview) and build your first Teams app.\nYou can create it from the scratch or explore our samples to help you quickly get started with the real-world examples and code structures.\n[Create a new Teams app](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22WalkThrough%22%5D)",
-    "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.title": "Build your first Teams app",
+    "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.description": "Start with [Teams](https://aka.ms/teamsfx-capabilities-overview) or [Outlook add-in](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview) capability to build your first app.\nYou can create it from the scratch or explore our samples to help you quickly get started with the real-world examples and code structures.\n[Create a new app](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22WalkThrough%22%5D)",
+    "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.title": "Build your first app",
```

Comment:
Let's also change outlook add-in capability to aka link in case we need to update it. Here it is: https://aka.ms/teamsfx-outlook-add-in-capabilities